static/js/mobile.js

```diff
@@ -0,0 +1,75 @@
+var loadingMessage = document.getElementById('msg');
+
+var useLoc = document.getElementById('use-loc');
+useLoc.checked = localStorage.useLoc === 'true';
+useLoc.onchange = function() {
+	localStorage.useLoc = useLoc.checked;
+};
+
+var tm;
+var useTimer = document.getElementById('use-timer');
+useTimer.checked = localStorage.useTimer !== 'false';
+useTimer.onchange = function() {
+	localStorage.useTimer = useTimer.checked;
+	if (localStorage.useTimer === 'false') {
+		clearTimeout(tm);
```

Comment:
__[eslint]__
* Error - 'clearTimeout' is not defined. (no-undef)

<!-- Comment by [SideCI](https://sideci.com) -->